recipe/meta.yaml

```
@@ -10,21 +10,23 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   disable_pip: true
   entry_points:
     - pip = pip._internal.cli.main:main
     - pip3 = pip._internal.cli.main:main
 
 requirements:
   host:
-    - python >=3.8
-    - setuptools
-    - wheel
+    - python >=3.13.a0           # [not with_setuptools_wheel]
+    - python >=3.8,<3.13.a0      # [with_setuptools_wheel]
+    - setuptools                 # [with_setuptools_wheel]
+    - wheel                      # [with_setuptools_wheel]
```

```suggestion
    - python >=3.8
    - setuptools
    - wheel
```

I just noticed this but don't we still need these in host, just not run?